src/components/layout/Folder/Folder.jsx

```diff
@@ -0,0 +1,18 @@
+import './folder.css';
+
+const Folder = ({ folder }) => {
+  const { name, owner } = folder;
```

Comment:
어차피 구조분해할당해서 사용할거라면 객체를 통으로 받지 않고 풀어진 형태로 받는건 어떠시나요?
```suggestion
const Folder = ({ name,owner }) => {
```

사용처에서는 아래와 같이 props 를 넘겨줄 수 있겠네요
```jsx
<Folder {...folder} />
```